src/schemas/HelmGitRepositoryResponse.yaml

```diff
@@ -0,0 +1,2 @@
+allOf:
```

Comment:
If ok for you @ctjhoa , I don't know if it produces weird structure on typescript generation, otherwise I can make a dedicated structure with all the fields (it's the same structure as for the Application git repo)

Reply:
(or I keep the `ApplicationGitRepositoryResponse` but don't find it good imo as it mixes many "domain")

Reply:
I think it's fine to keep `ApplicationGitRepositoryResponse` as it's already used in other service types
https://github.com/Qovery/qovery-openapi-spec/blob/main/src/schemas/job/BaseJobResponse.yaml#L81

Reply:
done, the change is easiest :smile: